pancancer-arch-3/src/main/java/info/pancancer/arch3/worker/WorkerRunnable.java

```diff
@@ -364,6 +350,34 @@ private WorkflowResult launchJob(String message, Job job, String seqwareEngine,
         return workflowResult;
     }
 
+    private String writeDockerRunnerScript(List<String> dockerArgs) throws IOException {
+
+        StringBuilder sb = new StringBuilder();
+        sb.append("#! /bin/bash\n");
+        sb.append("docker run ");
+        for (String arg : dockerArgs) {
+            sb.append(arg).append(" ");
+        }
+        sb.append(" && cat /home/" + this.userName + "/worker.cid >> /home/" + this.userName + "/successful_container_cids");
```

Comment:
I don't think apache exec can handle things like `some_command && cat some_file >> some_othe_file`, so it would have to be done in Java.

We don't want to append the CID to "successful_container_cids" if the `docker run...` command fails. It should only be appended if the `docker run...`  succeeds.